valueformatting.go

```diff
@@ -161,8 +163,56 @@ func (f *valueFormatting) binaryFormatter(
 	}
 }
 
+// Returns a valueFormatter function that pretty-prints JSON values.
```

Comment:
Godoc comments should begin with the name of the element it describes. So this should be `jsonFormatter returns ..."

Reply:
Done.